anthos-multi-cloud/Azure/vars.sh

```diff
@@ -0,0 +1,9 @@
+export CLUSTER_NAME=azure-cluster-fg
```

Comment:
Probably want to scrub personal project details from here.

Reply:
Yep, removed